Tools/TransformTool/TransformTool.cs

```diff
@@ -227,10 +229,11 @@ public void ProcessInput(ActionMapInput input, ConsumeControlDelegate consumeCon
 				if (manipulatorGameObject.activeSelf && hoveringSelection)
 					manipulatorGameObject.SetActive(false);
 
-				foreach (var selection in m_HoverObjects.Values)
+				foreach (var selection in m_HoverObjects)
```

Comment:
Remind me again why we highlight from the transform tool and not the selection tool? We can discuss live, too.